src/JsonPath/Expressions/ExistsLogicalExpressionNode.cs

```diff
@@ -29,6 +29,11 @@ public override bool Evaluate(JsonNode? globalParameter, JsonNode? localParamete
 
 		return result.Matches.Count != 0;
 	}
+	/// <inheritdoc/>
```

Comment:
`inheritdoc` historically has not worked well for me.  Please copy does from base.

Reply:
fixed